survey_form/index.html

```diff
@@ -43,8 +43,87 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form__survey--title">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label class="form-survey__label" for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
+
+      <label class="form-survey__label" for="email">Email: </label>
+      <input
+        type="email"
+        id="email"
+        class="form-survey__input"
+        required
+        placeholder="leonardo.dicaprio@gmail.com"
+      />
+
+      <label class="form-survey__label" for="number">Age: </label>
+      <input
+        type="number"
+        id="number"
+        class="form-survey__input"
+        placeholder="44"
+        min="12"
+        max="123"
+        required
+      />
+
+      <label class="form-survey__label" for="form-survey__dropdown"
```

Comment:
Mechanizm, który łączy input z labelem działa poprzez porównywanie atrybutu `for` w labelu z atrybutem `id` w inpucie. Także połącz odpowiednio ten label z selectem.